### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by DTCurrie**

### CODE REVIEW

**Review by micheal-parks — APPROVED** [review]

**Review by mcous — APPROVED** [review]
SHould we also set peer dependency resolution to strict in the root `.npmrc`? See viamrobotics/app#2740

**Comment by DTCurrie** [comment]
> SHould we also set peer dependency resolution to strict in the root `.npmrc`? See [viamrobotics/app#2740](https://github.com/viamrobotics/app/pull/2740)

Makes sense to me, will add it.

File: packages/sdk-svelte/.npmrc
Comment (mcous) [inline packages/sdk-svelte/.npmrc:1]: Pretty sure this should be in the root `.npmrc`, not in `packages/sdk-svelte`

Comment (DTCurrie) [reply packages/sdk-svelte/.npmrc:1]: There is no root `.npmrc`, so I will just make this the `root` one.

**Comment by DTCurrie** [comment]
> > SHould we also set peer dependency resolution to strict in the root `.npmrc`? See [viamrobotics/app#2740](https://github.com/viamrobotics/app/pull/2740)
> 
> Makes sense to me, will add it.

So I actually took it out because one issue. I want to use our svelte eslint/prettier configs but they rely on `eslint-plugin-tailwindcss`, which itself has a peer dep on `tailwindcss`. We can just install them, but we don't and probably won't use them in this project. I tried to extend the configs to just not that plugin, but it was not working. Might be worth looking into for when we want svelte libraries that don't include presentational components.

**Pull request merged**